src/qt/walletcontroller.cpp

```diff
@@ -207,6 +207,9 @@ void WalletControllerActivity::showProgressDialog(const QString& label_text)
     m_progress_dialog->setCancelButton(nullptr);
     m_progress_dialog->setWindowModality(Qt::ApplicationModal);
     GUIUtil::PolishProgressDialog(m_progress_dialog);
+    // The setValue call forces QProgressDialog to start the internal duration estimation.
+    // See details in https://bugreports.qt.io/browse/QTBUG-47042.
```

Comment:
In commit "qt: Improve GUI responsiveness" (4935ac583bbdc289dd31a1caae3d711edef742b6)

It could be useful to also add this comment other places we're relying on this behavior (or maybe dedup the code more generally):

https://github.com/bitcoin/bitcoin/blob/860093401840d7aad7b439aeba0d1598933bc9c6/src/qt/bitcoingui.cpp#L1380
https://github.com/bitcoin/bitcoin/blob/860093401840d7aad7b439aeba0d1598933bc9c6/src/qt/walletview.cpp#L335

Reply:
@ryanofsky

Thank you for your review.

I agree with you that there is a possibility to dedup the code more generally. As such a change must be refactoring, I intentionally leaved it for a follow up, if there are no objections from you.

A comment added here because this dialog is not actually a "progress" one, it is a busy indicator without meaningful values. But it requires to call `setValue(0)` to start [internal timer](https://github.com/bitcoin-core/gui/pull/343#discussion_r640163408).

In other cases calling `setValue()` is natural, as dialogs are real "progress" ones, and the value shows the progress.

Reply:
Right, the progress here is [0,0].